### DISCUSSION TITLE
Make themed banners replace default queue banner

### DISCUSSION THREAD

**Discussion by sph432** [OpenRCT2/OpenRCT2#15780] [Feature requests]
- [x] Reproducible in RCT2 (vanilla)
- [ ] Specific to multiplayer

**Steps to reproduce:**
1. Construct a ride, any ride. Place entrance and exit. Connect queue path.
2. Put any banners on the end of the queue over the one already there. Notice the duplicate banner. Depending on the view angle and the style of the placed banner, there might even be a nasty Z-fight:
![20191218_223735](https://user-images.githubusercontent.com/313648/71125477-478ff480-21e7-11ea-96a3-ab153835bd80.png)

**Comment by ocalhoun6** [comment]
Is it feasible to make the banner automatically assume what the ride's entrance banner would normally be saying, including changing to -- Closed -- when the ride is closed and updating when the ride's name is changed?

**Comment by duncanspumpkin** [comment]
No it is not feasible. This looks like new save territory.
